src/calculateRentalCost.js

```diff
@@ -27,8 +27,16 @@
  * @return {number}
  */
 
+const everyDayCost = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 2) {
+    return everyDayCost * days;
+  } else if (days >= 3 && days <= 6) {
+    return everyDayCost * days - 20;
+  } else if (days >= 7) {
```

Comment:
```suggestion
  if (days <= 2) {
    return everyDayCost * days;
  }

  if (days <= 6) {
    return everyDayCost * days - 20;
  }
```
You can omit else, since each if has return statement inside